ui/src/scss/_patterns_icons.scss

```diff
@@ -30,7 +30,7 @@
 
   .p-icon--locked {
     @extend %icon;
-    @include maas-icon-power($color-mid-dark);
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='16' width='12'%3E%3Cpath d='M6 0C3.793 0 2 1.793 2 4v3H0v9h12V7h-2V4c0-2.207-1.793-4-4-4zm0 1c1.67 0 3 1.33 3 3v3H3V4c0-1.67 1.33-3 3-3zm1 8.5V14H5v-4l2-.5z' fill='%23888' fill-rule='evenodd'/%3E%3C/svg%3E");
```

Comment:
I wonder if naming a icon mixin would be more clear and help make icons more manageable in the project. Would also be consistent with the use of the removed icon mixin.

Reply:
Good call! I've added a `maas-icon-lock` mixin and included it here